model/model.go

```diff
@@ -404,9 +404,17 @@ type ViewInfo struct {
 	Algorithm   ViewAlgorithm      `json:"view_algorithm"`
 	Definer     *auth.UserIdentity `json:"view_definer"`
 	Security    ViewSecurity       `json:"view_security"`
-	SelectStmt  string             `json:"view_select"`
+	SelectStmt  []string           `json:"view_select"`
```

Comment:
We need a detailed description here for
1. why we need to store strings here
2. what do they store individually